src/routes/docs/advanced/graphql.mdx

```diff
@@ -98,6 +98,16 @@ There are a lot more things Mirage GraphQL can do. For example, you can
 
 and more.
 
-Mirage GraphQL is v0.1.*x*, but extracted from years of work on the `ember-cli-mirage-graphq` library.
+Mirage GraphQL is v0.1.*x*, but extracted from years of work on the `ember-cli-mirage-graphql` library.
 
 To learn more, check out the README of [Mirage GraphQL](https://github.com/miragejs/graphql). Also check out the intergration tests for more in-depth examples of options, queries and mutations.
+
+## Mirage & `graphql-mocks`
+
+As an unofficial community-based alternative to mocking with Mirage and GraphQL check out [`graphql-mocks`](https://www.graphql-mocks.com) and its [Mirage middleware](https://www.graphql-mocks.com/docs/guides/mirage-js).
+
+Consider using `graphql-mocks` with Mirage for:
+* Composable middlewares for managing and creating different scenarios and states
+* Resolver and resolver map abstractions for extending query resolutions
+* Agnostic network and support for Node
```

Comment:
I don't quite understand this point and I'm not sure how to suggest anything helpful